etc/config.xml

```diff
@@ -31,6 +31,7 @@ productalert</exclude_paths>
             </bfcache>
             <view_transitions>
                 <enable>1</enable>
+                <enable_admin>0</enable_admin>
```

Comment:
IMO it should be enabled by default, given (I suspect) a significant proportion of installs will not be using the Mage-OS or Hyva admin theme.

```suggestion
                <enable_admin>1</enable_admin>
```